client/blocks/signup-form/index.jsx

```diff
@@ -1088,6 +1088,18 @@ class SignupForm extends Component {
 			return null;
 		}
 
+		if ( this.props.currentUser && ! this.props.disableContinueAsUser ) {
```

Comment:
Moving this before the next condition ensures it's displayed for Crowdsignal too